templates/types/streaming/fastapi/app/settings.py

```diff
@@ -88,9 +91,30 @@ def init_azure_openai():
     Settings.embed_model = AzureOpenAIEmbedding(**embedding_config)
 
 
+def init_groq():
+    from llama_index.embeddings.huggingface import HuggingFaceEmbedding
+    from llama_index.llms.groq import Groq
+
+    model_map: Dict[str, str] = {
+        "llama3-8b": "llama3-8b-8192",
```

Comment:
@Jac-Zac do we need this model map for Groq in Python? e.g. i can't use `llama3-8b` directly in Groq in Python?
Ideally, we won't have this map as in TS (see `settings.ts`)

(note: for hugging face embedding, we need the mapping)

Reply:
I think we need it since those are the models that are provided by the groq api: https://console.groq.com/docs/models

Reply:
I'm not sure in which exactly, I took inspiration from the anthropic one since I think it quite similar actually. Also I have added a commit which I think might be needed, we can also squash it later

Reply:
@Jac-Zac I added a comment where to put the lookup